src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/198043270-6c337f29-b7bf-49d7-9598-8f2a2d222842.png)

3 is a magic number